nodejs/week3/meal-sharing/src/backend/api/reservations.js

```diff
@@ -0,0 +1,80 @@
+const express = require("express");
+const router = express.Router();
+const knex = require("../database");
+
+//...........................................Returns all meals	
+
+router.get("/", async (request, response) => {
+
+    try {
+        // knex syntax for selecting things. Look up the documentation for knex for further info
+        const resault = await knex("reservations").select("*");
+        response.json(resault);
+
+    } catch (error) {
+        throw error;
+    }
+});
+
+//............................................Adds a new reservation
+
+router.post("/", async (request, response) => {
+
+    try {
+        // knex syntax for selecting things. Look up the documentation for knex for further info
+        const resault = await knex("reservations").insert(request.body);
```

Comment:
It's probably a good idea to name the variable something that resembles what you expect to be in the variable, so `reservations` in this example.